dev.sh

```diff
@@ -2,6 +2,8 @@
 set -e
 VENV=./venv
 
+pip install -U virtualenv || sudo pip install -U virtualenv
```

Comment:
upgrade is not necessary - but why not?
install is necessary - travis does not ship with virtualenv.